pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1515,6 +1597,24 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListenerForPortOrUDS(n
 
 			listenerOpts.filterChainOpts = opts
 
+		case istionetworking.ListenerProtocolDubbo:
```

Comment:
also skip

Reply:
Unlike mysql/mangodb, Dubbo filter handles the routing by itself instead of passing it through to a tcp proxy filter.